python/lsst/daf/butler/instrument.py

```diff
@@ -96,12 +96,11 @@ def makeExposureEntryFromVisitInfo(dataId, visitInfo, snap=0):
     """
     avg = visitInfo.getDate()
```

Comment:
The docstring above should say `lsst.afw.image.VisitInfo` and `lsst.afw.image.Exposure`

Reply:
:+1: for `lsst.afw.image.VisitInfo`, but this is a daf_butler Exposure DataUnit, not an `lsst.afw.image.Exposure`...I need to follow up on that rename.